airbyte-commons-server/src/test/java/io/airbyte/commons/server/handlers/ConnectorBuilderProjectsHandlerTest.java

```diff
@@ -97,6 +108,26 @@ void testUpdateConnectorBuilderProject() throws IOException, JsonValidationExcep
             project);
   }
 
+  @Test
+  @DisplayName("updateConnectorBuilderProject should update an existing project removing the draft")
+  void testUpdateConnectorBuilderProjectWipeDraft() throws IOException, ConfigNotFoundException {
+    final ConnectorBuilderProject project = generateBuilderProject();
+    project.setManifestDraft(null);
+
+    when(configRepository.getConnectorBuilderProject(project.getBuilderProjectId(), false)).thenReturn(project);
+
+    final ExistingConnectorBuilderProjectWithWorkspaceId update = new ExistingConnectorBuilderProjectWithWorkspaceId()
+        .builderProject(new ConnectorBuilderProjectDetails().name(project.getName()))
+        .workspaceId(workspaceId).builderProjectId(project.getBuilderProjectId());
+
+    connectorBuilderProjectsHandler.updateConnectorBuilderProject(update);
+
+    verify(uuidSupplier, never()).get();
+    verify(configRepository, atMostOnce())
```

Comment:
why `atMostOnce()` instead of exactly once?

Reply:
should be exactly once, fixed